chatty-clownfish/bangazon-sprint-2
# Dc ticket#14productdetails

## Link to Ticket
Closes # [Link]()

## Description of Proposed Changes
This PR should be able to 
- 

## Steps to Test

Outline the steps to test


1. go to the products page. Each product should be a link to the detail page for each product.

2. once going to details page you should see the description, title, quantity, and price of each product.

3. you should see a add to card button. this button does not have any functionality yet and will explode when you press it.

```sh
git fetch --all
git checkout dc-ticket#14productdetails
```


## Impacted Areas in Application

List general components of the application that this PR will affect:
This PR does affect the products page as well as the products detail page.
